.github/workflows/release.yml

```
@@ -24,12 +24,16 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Install Helm
-        uses: azure/setup-helm@v3
+        uses: azure/setup-helm@v3.5
+        with:
+          version: v3.12.3
```

I'm a bit confused here. Couldn't we simply use the following which would use the latest v3.12.x?

```suggestion
        uses: azure/setup-helm@v3.12
```

---

@gabor-boros the action version doesn't match the Helm version. This change was inspired by what prometheus-community charts do for their [releases](https://github.com/prometheus-community/helm-charts/blob/main/.github/workflows/release.yaml#L28)

---

Oh! Now I see! Thank you!